### PR TITLE
remove $ form facet search

### PR DESCRIPTION
**Pull request by dms1lva**
Using shift + del while in facet search excludes the currently selected value. JQuery was still used to do this.

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)
https://coveord.atlassian.net/browse/JSUI-1146

### CODE REVIEW

File: src/ui/Base/CoveoJQuery.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ interface IWindow {
 }
 
 export var jQueryInstance: JQueryStatic;
+export var swag: Object;
```
Comment (olamothe) [inline src/ui/Base/CoveoJQuery.ts]: Huh ?  :)

File: src/ui/Facet/ValueElement.ts [hunk 2]
```diff
@@ -64,6 +65,17 @@ export class ValueElement {
     this.facetValue.selected = false;
     this.facetValue.excluded = true;
     this.renderer.setCssClassOnListValueElement();
+    var actionCause: IAnalyticsActionCause;
```
Comment (olamothe) [inline src/ui/Facet/ValueElement.ts]: Why did you move this from the click action ?

It's intentional that the code was in the click action, since we're tracking end user interaction with the search interface, not javascript code that calls exclude or select on values.

File: src/ui/Facet/ValueElement.ts [hunk 3]
```diff
@@ -45,6 +45,7 @@ export class ValueElement {
       this.handleEventForExcludedValueElement(eventBindings);
     } else {
       this.handleEventForValueElement(eventBindings);
+      $();
```
Comment (olamothe) [inline src/ui/Facet/ValueElement.ts]: ??
Shouldn't this fail on compilation ?

Comment (dms1lva) [inline src/ui/Facet/ValueElement.ts]: Yeah was trying to figure out why it wasn't failing on compilation and forgot :(

**Comment by olamothe** [comment]
:shipit:

**Comment by francoislg** [comment]
Renaming `JQueryStatic` and removing the ambient jQuery definition file required us to refer to that ambiant file ourselves from the 1.1550 release to the 1.1865 January release.

* `JQueryStatic` is no longer available
* Global `jQuery` instance is no longer available
* `JQueryEventObject` should now be `Event`
* Maybe others, but this is what affected us.

It should probably be noted in the Release Notes that you removed ambient jQuery and that some upgrade may be required with this version.
